### ISSUE TITLE
inconsistent line separators for some files

### ISSUE THREAD

**Issue by SunBeomSo** [smartbugs/smartbugs#41]
In some files, lines are separated by ``\r\n``, rather than ``\n``.

For example, in Python2 (in Python3, it seems ``\r\n`` is automatically converted to ``\n`` when using ``readlines()`` function),
you can observe that each line is separated by ``\r\n`` for the file https://github.com/smartbugs/smartbugs/blob/master/dataset/reentrancy/0x01f8c4e3fa3edeb29e514cba738d87ce8c091d3f.sol:
```
with open (‘0x01f8c4e3fa3edeb29e514cba738d87ce8c091d3f.sol’, ‘r’) as fp:
  fp.readlines()
```
which has the following result:
```
['/*\r\n', ' * @source: etherscan.io \r\n', ' * @author: -\r\n', ' * @vulnerable_at_lines: 54\r\n', ' */\r\n', '\r\n', 'pragma solidity ^0.4.19;\r\n', '\r\n', 'contract PERSONAL_BANK\r\n', '{\r\n', '    mapping (address=>uint256) public balances;   \r\n', '   \r\n', '    uint public MinSum = 1 ether;\r\n', '    \r\n', '    LogFile Log = LogFile(0x0486cF65A2F2F3A392CBEa398AFB7F5f0B72FF46);\r\n', '    \r\n', '    bool intitalized;\r\n', '    \r\n', '    function SetMinSum(uint _val)\r\n', '    public\r\n', '    {\r\n', '        if(intitalized)revert();\r\n', '        MinSum = _val;\r\n', '    }\r\n', '    \r\n', '    function SetLogFile(address _log)\r\n', '    public\r\n', '    {\r\n', '        if(intitalized)revert();\r\n', '        Log = LogFile(_log);\r\n', '    }\r\n', '    \r\n', '    function Initialized()\r\n', '    public\r\n', '    {\r\n', '        intitalized = true;\r\n', '    }\r\n', '    \r\n', '    function Deposit()\r\n', '    public\r\n', '    payable\r\n', '    {\r\n', '        balances[msg.sender]+= msg.value;\r\n', '        Log.AddMessage(msg.sender,msg.value,"Put");\r\n', '    }\r\n', '    \r\n', '    function Collect(uint _am)\r\n', '    public\r\n', '    payable\r\n', '    {\r\n', '        if(balances[msg.sender]>=MinSum && balances[msg.sender]>=_am)\r\n', '        {\r\n', '            // <yes> <report> REENTRANCY\r\n', '            if(msg.sender.call.value(_am)())\r\n', '            {\r\n', '                balances[msg.sender]-=_am;\r\n', '                Log.AddMessage(msg.sender,_am,"Collect");\r\n', '            }\r\n', '        }\r\n', '    }\r\n', '    \r\n', '    function() \r\n', '    public \r\n', '    payable\r\n', '    {\r\n', '        Deposit();\r\n', '    }\r\n', '    \r\n', '}\r\n', '\r\n', '\r\n', '\r\n', 'contract LogFile\r\n', '{\r\n', '    struct Message\r\n', '    {\r\n', '        address Sender;\r\n', '        string  Data;\r\n', '        uint Val;\r\n', '        uint  Time;\r\n', '    }\r\n', '    \r\n', '    Message[] public History;\r\n', '    \r\n', '    Message LastMsg;\r\n', '    \r\n', '    function AddMessage(address _adr,uint _val,string _data)\r\n', '    public\r\n', '    {\r\n', '        LastMsg.Sender = _adr;\r\n', '        LastMsg.Time = now;\r\n', '        LastMsg.Val = _val;\r\n', '        LastMsg.Data = _data;\r\n', '        History.push(LastMsg);\r\n', '    }\r\n', '}\r\n']
```

On the other hand,  you can observe that each line is separated by “\n” (expected one to me) for the file https://github.com/smartbugs/smartbugs/blob/master/dataset/reentrancy/reentrance.sol :
```
with open (‘reentrance.sol’, ‘r’) as fp:
  fp.readlines()
```
which has the following result:
```
['/*\n', ' * @source: https://ethernaut.zeppelin.solutions/level/0xf70706db003e94cfe4b5e27ffd891d5c81b39488\n', ' * @author: Alejandro Santander\n', ' * @vulnerable_at_lines: 24\n', ' */\n', '\n', 'pragma solidity ^0.4.18;\n', '\n', 'contract Reentrance {\n', '\n', '  mapping(address => uint) public balances;\n', '\n', '  function donate(address _to) public payable {\n', '    balances[_to] += msg.value;\n', '  }\n', '\n', '  function balanceOf(address _who) public view returns (uint balance) {\n', '    return balances[_who];\n', '  }\n', '\n', '  function withdraw(uint _amount) public {\n', '    if(balances[msg.sender] >= _amount) {\n', '      // <yes> <report> REENTRANCY\n', '      if(msg.sender.call.value(_amount)()) {\n', '        _amount;\n', '      }\n', '      balances[msg.sender] -= _amount;\n', '    }\n', '  }\n', '\n', '  function() public payable {}\n', '}\n']
```

In my experience, solc (with an option ``--ast-compact-json``) generates different source-to-bytes information when the only difference between the two files is the line separator.

However, the byte-related module in OCaml (and potentially for other languages, too; let's consider the above case when using Python3) seems to produce consistent results no matter what the line separators are (I guess ``\r\n`` is automatically converted to ``\n``).
 
As a result, tools that compute line numbers based on solc's source-to-bytes information may report numbers incorrectly, even when the tools correctly detect bugs in their own intermediate representations.

In summary, I think line separators should be consistent for all files (as ``\n`` rather than ``\r\n``).
